kv/main.go

```diff
@@ -129,7 +131,7 @@ type generator struct {
 func newGenerator(seq *sequence) *generator {
 	return &generator{
 		seq:    seq,
-		rand:   rand.New(rand.NewSource(int64(time.Now().UnixNano()))),
+		rand:   rand.New(rand.NewSource(rand.Int63())),
```

Comment:
Good point about seeding the global rand, I've indeed set us up for a problem here. I'll back this out (the concern above is synthetic anyway).